server/channels/api4/integration_action_test.go

```diff
@@ -41,7 +41,8 @@ func (th *testHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	assert.NotEmpty(th.t, poir.TriggerId)
 	assert.Equal(th.t, "button", poir.Type)
 	assert.Equal(th.t, "test-value", poir.Context["test-key"])
-	w.Write([]byte("{}"))
+	_, err = w.Write([]byte("{}"));
```

Comment:
Let's remove that semicolon

```suggestion
	_, err = w.Write([]byte("{}"))
```